sdk/maps/azure-maps-search/azure/maps/search/_generated/models/_enums.py

```diff
@@ -0,0 +1,179 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from enum import Enum
+from azure.core import CaseInsensitiveEnumMeta
+
+
+class BoundaryResultTypeEnum(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
Please remove the `Enum` suffix from all these enum names.

Reply:
We are ok with modifying the generated files?

Reply:
No - this should be fixed in the spec/codegen config.